zerver/webhooks/travis/tests.py

```diff
@@ -57,6 +57,36 @@ def test_travis_pull_requests_are_not_ignored_when_applicable(self) -> None:
             content_type="application/x-www-form-urlencoded",
         )
 
+    def test_travis_only_push_event(self) -> None:
+        self.subscribe(self.test_user, self.STREAM_NAME)
+        self.url = f'{self.build_webhook_url()}&only_events=["push"]'
+
+        expected_message = (
+            "Author: josh_mandel\nBuild status: Passed :thumbs_up:\n"
+            "Details: [changes](https://github.com/hl7-fhir/fhir-sv"
+            "n/compare/6dccb98bcfd9...6c457d366a31), [build log](ht"
+            "tps://travis-ci.org/hl7-fhir/fhir-svn/builds/92495257)"
+        )
```

Comment:
Let's write this test how most other webhook tests are, namely with `"""` quote blocks so that one can see the newlines naturally; it'll be a lot more readable.

(Maybe worth redoing this file in that style as a prep commit)